src/gcpsample_client/publishing.py

```diff
@@ -42,21 +42,6 @@
 video_spec = DigitalVideoSpecification.create()
 video_spec.displayNameTrait().setName("Ginos Home Video")
 # NOTE: It is critical to always use the working_ref from now on.
-working_ref = manager.preflight(entity_ref, video_spec, context)
-
-# We then check if the manager can tell us where to save the file.
-if ResolvesFutureEntitiesTrait.isImbuedTo(policy):
-    working_data = manager.resolve(
-            working_ref, DigitalVideoSpecification.kTraitSet, context)
-    working_spec = DigitalVideoSpecification(working_data)
-    # GF: just print the display name
-    print(f"This video is called: {working_spec.displayNameTrait().getName}")
-
-    # GF: this is the old code for files, not needed for now
-    # if save_url := working_spec.locatableContentTrait().getLocation():
-    #     save_path = pathFromURL(save_url)  # URL decode etc
-    # if custom_encoding := working_spec.textEncodingTrait().getEncoding():
-    #     encoding = custom_encoding
```

Comment:
You only need to use [`preflight`](https://openassetio.github.io/OpenAssetIO/classopenassetio_1_1v1_1_1host_api_1_1_manager.html#a5310cd7307fbdf5cb15998129a6f3336) if you're about to do some work to determine the rest of the data that is needed for the publish. If you already know everything (eg, in this case, the name of the video), then you don't need to call it, and you can just a`register` all the data for the entity.

Reply:
Understood.... but I'm going to leave it in there as a no-op and make a few comments to myself.